webui/react/src/hooks/useFeature.ts

```diff
@@ -1,13 +1,30 @@
+import { map } from 'fp-ts/lib/Record';
+import { boolean, null as ioNull, keyof, partial, TypeOf, union } from 'io-ts';
 import { useObservable } from 'micro-observables';
 
 import determinedStore, { DeterminedInfo } from 'stores/determinedInfo';
+import userSettings from 'stores/userSettings';
+import { Loadable } from 'utils/loadable';
 
-// Add new feature switches below using `|`
-export type ValidFeature = 'dashboard' | 'explist_v2' | 'chart' | 'rp_binding';
+// add new feature switches here
+export const FEATURES = {
+  chart: 'New Charts',
```

Comment:
'Enable improved learning curve charts for experiment visualizations'
'Enable improved experiment listing, filtering, and comparison'
'Allow resource pools to be assigned to workspaces'